utils/utils.tsx

```diff
@@ -123,26 +134,18 @@ export function isKeyPressed(event, key) {
     return event.keyCode === key[1];
 }
 
-/**
- * check keydown event for line break combo. Should catch alt/option + enter not all browsers except Safari
- * @param  {object}  e - keydown event
- * @return {boolean}
- */
-export function isUnhandledLineBreakKeyCombo(e) {
+// check keydown event for line break combo. Should catch alt/option + enter not all browsers except Safari
+export function isUnhandledLineBreakKeyCombo(e: React.KeyboardEvent | KeyboardEvent): boolean {
     return Boolean(
         isKeyPressed(e, Constants.KeyCodes.ENTER) &&
         !e.shiftKey && // shift + enter is already handled everywhere, so don't handle again
         (e.altKey && !UserAgent.isSafari() && !cmdOrCtrlPressed(e)), // alt/option + enter is already handled in Safari, so don't handle again
     );
 }
 
-/**
- * insert a new line character at keyboard cursor (or overwrites selection)
- * @param  {object}  e - keydown event
- * @return {string} message modified with new line inserted for app consumption
- * WARNING: HAS DOM SIDE EFFECTS
- */
-export function insertLineBreakFromKeyEvent(e) {
+// insert a new line character at keyboard cursor (or overwrites selection)
+// WARNING: HAS DOM SIDE EFFECTS
+export function insertLineBreakFromKeyEvent(e: React.KeyboardEvent | any): string {
```

Comment:
Why add any as an option here?

Reply:
not needed. specified that it should be an input or text area and then asserted in the callers that they were.